internal/utils.go

```diff
@@ -139,12 +140,24 @@ func getPolicyTemplates(policyConf *types.PolicyConfig) ([]map[string]map[string
 	for i, manifestGroup := range manifestGroups {
 		complianceType := policyConf.Manifests[i].ComplianceType
 		for _, manifest := range manifestGroup {
+			isPolicyTypeManifest, err := isPolicyTypeManifest(manifest)
+			if err != nil {
+				return nil, err
```

Comment:
Could you add additional context to the error so the user knows which manifest has the invalid `apiVersion` or `kind`?